vnext/local-cli/runWindows/utils/commandWithProgress.js

```diff
@@ -50,18 +71,23 @@ function commandWithProgress(spinner, taskDoingName, command, args, verbose) {
     }
 
     const cp = child_process.spawn(command, args, spawnOptions);
-
+    var error = null;
```

Comment:
`var` shouldn't be used in modern JS, since it has confusing behavior around function local scope (I've had a real bug from this before). Use `let` instead here (or `const` if it could be).